api/api.go

```diff
@@ -80,27 +80,31 @@ const (
 	// SpecMatchSrcVolProvision defaults to false. Applicable to cloudbackup restores only.
 	// If set to "true", cloudbackup restore volume gets provisioned on same pools as
 	// backup, allowing for inplace restore after.
-	SpecMatchSrcVolProvision   = "match_src_vol_provision"
-	SpecNodiscard              = "nodiscard"
-	StoragePolicy              = "storagepolicy"
-	SpecCowOnDemand            = "cow_ondemand"
-	SpecDirectIo               = "direct_io"
-	SpecScanPolicyTrigger      = "scan_policy_trigger"
-	SpecScanPolicyAction       = "scan_policy_action"
-	SpecProxyWrite             = "proxy_write"
-	SpecSharedv4ServiceType    = "sharedv4_svc_type"
-	SpecSharedv4ServiceName    = "sharedv4_svc_name"
-	SpecSharedv4ExternalAccess = "sharedv4_external_access"
-	SpecFastpath               = "fastpath"
-	SpecAutoFstrim             = "auto_fstrim"
-	SpecBackendType            = "backend"
-	SpecBackendPureBlock       = "pure_block"
-	SpecBackendPureFile        = "pure_file"
-	SpecPureFileExportRules    = "pure_export_rules"
-	SpecIoThrottleRdIOPS       = "io_throttle_rd_iops"
-	SpecIoThrottleWrIOPS       = "io_throttle_wr_iops"
-	SpecIoThrottleRdBW         = "io_throttle_rd_bw"
-	SpecIoThrottleWrBW         = "io_throttle_wr_bw"
+	SpecMatchSrcVolProvision        = "match_src_vol_provision"
+	SpecNodiscard                   = "nodiscard"
+	StoragePolicy                   = "storagepolicy"
+	SpecCowOnDemand                 = "cow_ondemand"
+	SpecDirectIo                    = "direct_io"
+	SpecScanPolicyTrigger           = "scan_policy_trigger"
+	SpecScanPolicyAction            = "scan_policy_action"
+	SpecProxyWrite                  = "proxy_write"
+	SpecSharedv4ServiceType         = "sharedv4_svc_type"
+	SpecSharedv4ServiceName         = "sharedv4_svc_name"
+	SpecSharedv4FailoverDelay       = "sharedv4_failover_delay"
+	SpecSharedv4FailoverDelayNormal = "normal"
```

Comment:
Instead of `normal` and `short`, we can use `conservative` and `aggressive` instead.

When selecting the two, the end user is picking a strategy instead of the actual duration.

@adityadani wdyk?

Reply:
In that case, we have the following options:
Option 1: A bit verbose but self explanatory
```
sharedv4_failover_delay_strategy: conservative | aggressive
```
Option 2: With the shorter version of key it is not explicit since they key `sharedv4_failover_delay` implies that you are setting some kind of time period
```
sharedv4_failover_delay: conservative | aggressive
```
Option 3: The original one
```
sharedv4_failover_delay: short | normal
```

I would vote for Option 1.

Reply:
One advantage of using "strategy" is that we could hide more settings behind it in the future. If that is the intention behind using the strategy, we should keep the name of the option generic and not include "delay" in the name. Also, we should keep "normal" as one of the two values so that we can add "conservative" as the third option in the future.

So maybe something like:
```
sharedv4_failover_strategy: aggressive | normal
```

I would still prefer the "sharedv4_failover_delay" option since it is crisp and clear and the values are shorter. We can still add other options as separate parameters in the future. But I am fine with going with the strategy option if you guys prefer that.

Reply:
Let's go ahead with `sharedv4_failover_strategy: aggressive | normal`